package.json

```diff
@@ -4,13 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --config webpack.config.js"
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
- [ ] The start script it's outdated. We need to user the newer version unless it will not work
`"start": "webpack serve --mode development --open"`